src/utils/validation.js

```diff
@@ -0,0 +1,30 @@
+const SUPPORTED_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function validateInput(originalText, targetCase) {
+  const validationMessages = [];
+
+  if (!targetCase) {
+    validationMessages.push({
+      message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
```

Comment:
Create constants for error messages. Also DRY, this message is repeated twice `Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"` create separate variable.